templates/components/common/header.html

```diff
@@ -16,13 +16,14 @@
         <h1 class="header-logo header-logo--{{theme_settings.logo-position}}">
             {{> components/common/store-logo}}
         </h1>
+        
+        {{{region name="homepage_banner"}}}
```

Comment:
I thought we were going to use header_bottom for this.
https://docs.google.com/spreadsheets/d/1P4ncQzCrCeJJp8sXgFuAxWKuDqhvFeHmkuJ3b5MytmI/edit#gid=0

Can't you just move it out of the conditional block to include this region on every page?